seolondon/settings/__init__.py

```diff
@@ -22,6 +23,8 @@
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/
 ALLOWED_HOSTS = []
+if os.environ.get('SEO_ALLOWED_HOST', None):
+    ALLOWED_HOSTS.append(os.environ.get('SEO_ALLOWED_HOST'))
```

Comment:
wondering why we need this here at all as it is set in the production settings, is it being used in some way for dev?